keras_microscopy/preprocessing/standardize.py

```diff
@@ -0,0 +1,71 @@
+import keras
+import numpy
+import skimage.exposure
+import skimage.restoration
+import skimage.transform
+
+
+def desaturate(ratio):
+    """
+    Fades the image to white. Higher ratio means more fading.
+    :param ratio: The ratio to fade by, between 0 and 1.
+    :return: The desaturate function.
+    """
+    return lambda x: x + (x.max() - x) * ratio
+
+
+def rescale(scale, **kwargs):
+    """
+    Rescales the image according to the scale ratio.
+    :param scale: The scalar to rescale the image by.
+    :param kwargs: Additional arguments for skimage.transform.resize.
+    :return: The rescale function.
+    """
+    if keras.backend.image_data_format() == 'channels_first':
+        return lambda x: skimage.transform.resize(x, (len(x[:, 0, 0]), len(x[0, :, 0]) * scale, len(x[0, 0, :]) * scale), **kwargs)
+    else:
+        return lambda x: skimage.transform.resize(x, (len(x[:, 0, 0]) * scale, len(x[0, :, 0]) * scale, len(x[0, 0, :])), **kwargs)
```

Comment:
Suggestion:

Depending on the channel order, use `numpy.multiply` to determine the scale factors. E.g.,

```python
if keras.backend.image_data_format() == 'channels_first':
    axes_scale = (1.0, scale, scale) 
else:
    axes_scale = (scale, scale, 1.0)

return skimage.transform.resize(x, numpy.multiply(x.shape, axes_scale))
```